examples/docker-compose/nginx/nginx.conf

```diff
@@ -4,69 +4,56 @@
 # https://botpress.com/docs/advanced/hosting/#secure-configuration-for-the-nginx-server
 ############################################################################################
 
-events {
-  worker_connections 1024;
-}
-
-http {
-  upstream botpress_server {
-    server botpress:3000;
-  }
-
-  # Disable sending the server identification
-  server_tokens off;
 
-  # Prevent displaying Botpress in an iframe (clickjacking protection)
-  add_header X-Frame-Options SAMEORIGIN;
+upstream botpress_server {
+  server botpress:3000;
+}
 
-  # Prevent browsers from detecting the mimetype if not sent by the server.
-  add_header X-Content-Type-Options nosniff;
+# Disable sending the server identification
+server_tokens off;
 
-  # Force enable the XSS filter for the website, in case it was disabled manually
-  add_header X-XSS-Protection "1; mode=block";
+# Prevent displaying Botpress in an iframe (clickjacking protection)
+add_header X-Frame-Options SAMEORIGIN;
 
-  # Configure the cache for static assets
-  proxy_cache_path /srv/nginx_cache levels=1:2 keys_zone=my_cache:10m max_size=10g
-                inactive=60m use_temp_path=off;
+# Prevent browsers from detecting the mimetype if not sent by the server.
+add_header X-Content-Type-Options nosniff;
 
-  # Set the max file size for uploads (make sure it is larger than the configured media size in botpress.config.json)
-  client_max_body_size 10M;
+# Force enable the XSS filter for the website, in case it was disabled manually
+add_header X-XSS-Protection "1; mode=block";
 
-  # Configure access
-  log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
-                    '$status $body_bytes_sent "$http_referer" '
-                    '"$http_user_agent" "$http_x_forwarded_for"';
+# Configure the cache for static assets
+proxy_cache_path /srv/nginx_cache levels=1:2 keys_zone=my_cache:10m max_size=10g
+              inactive=60m use_temp_path=off;
 
-  access_log  /access.log  main;
-  error_log  /error.log;
+# Set the max file size for uploads (make sure it is larger than the configured media size in botpress.config.json)
+client_max_body_size 10M;
```

Comment:
Maybe you should put 15M here since you specify in the comments that this value must be "larger than the configured media size in botpress.config.json"?